### PR TITLE
[compliance] add SPDX identifiers and scripts for adding SPDX.

### LINKED ISSUE #381: [compliance] add license identifiers to every .rs and .sh files
Like many other FOSS communities, e.g. Ethereum, we should provide SPDX IDs to every code files before releasing our code to the public.

There should be a bash script to do so, and CI to check and run the scripts.

SPDX Online Tool: https://tools.spdx.org/app/check_license/

### PR DESCRIPTION
**Pull request by feliciss**
resolve #381

### CODE REVIEW

File: crates/rooch-open-rpc/src/generate_json_rpc_spec.rs [hunk 1]
```diff
@@ -1,9 +1,6 @@
 // Copyright (c) RoochNetwork
 // SPDX-License-Identifier: Apache-2.0
 
-// Copyright (c) Mysten Labs, Inc.
```
Comment (jolestar) [inline crates/rooch-open-rpc/src/generate_json_rpc_spec.rs:4]: do not remote the original Copyright

**Review by jolestar — COMMENTED** [review]
Skip the *.move file.

Because the origin move_stdlib's move file does not add the license. We follow the community.

**Review by jolestar — COMMENTED** [review]
Can we auto-check the license in the CI?

**Comment by feliciss** [comment]
> Can we auto-check the license in the CI?

I think we can. I can write a script to check with the CI.

However, I do not possess the accessibility to the CI. So may someone to pick the CI up after the script is ready.

File: crates/rooch-open-rpc/src/generate_json_rpc_spec.rs (hunk 1 above)
Comment (feliciss) [reply crates/rooch-open-rpc/src/generate_json_rpc_spec.rs]: I leave it remained.

**Comment by feliciss** [comment]
I may add another script with things related to the CI check. You can review this PR and merge if it looks okay.

**Review by jolestar — APPROVED** [review]

**Pull request merged**